src/tests/Common/tests.targets

```diff
@@ -6,7 +6,7 @@
   </ItemGroup>
 
   <PropertyGroup>
-      <TestAssemblyDir Condition="'$(TestAssemblyDir)' == ''">$(BaseOutputPathWithConfig)\tests\</TestAssemblyDir>
+      <TestAssemblyDir Condition="'$(TestAssemblyDir)' == ''">$(BaseOutputPathWithConfig)\Tests\</TestAssemblyDir>
```

Comment:
Why does upstream not run into the same problem (in other words - is this an upstream bug)?

Reply:
Have reverted this one, it looked wrong, but wasn't the problem I thought it was.